Jenkinsfile.yaml

```diff
@@ -1,40 +1,60 @@
-gitCredentialId: dummyGitRepo
-artifactPublishingConfig:
-    host: 192.32.52.12
-    user: user53
-    credentialId: dummyGitRepo
+# same as defined in current version
+agent:
+#  label: my_label
+#  customWorkspace: path_to_workspace
+#  dockerImage: maven:3-alpine
+#  args: -v /tmp:/tmp
+  # so that tools will be a prt of agent only
+  # tools needs to be defined in the tool configuration
+  tools:
+    maven : maven_3.0.1
+    jdk : jdk8
 
-agent: any
+configuration:
+  # if true PR will be fetched and build will be started without merging it to the target branch
+  # else PR will be merged to the target branch and then build will be started.
+  # default value will be false
+  buildPROnly: true
```

Comment:
See comment below about abbreviating "PR" in a consistent way. Users will scratch their head if we have different spelling "PR" and "pr". Users need consistency.